src/app/shipping/Shipping.tsx

```diff
@@ -147,6 +147,31 @@ class Shipping extends Component<ShippingProps & WithCheckoutShippingProps, Ship
         );
     }
 
+    private handleMultiShippingModeSwitch: () => void = async () => {
+        const {
+            consignments,
+            isMultiShippingMode,
+            onToggleMultiShipping = noop,
+            onUnhandledError = noop,
+            updateShippingAddress,
+        } = this.props;
+
+        if (isMultiShippingMode && consignments.length > 1) {
+            this.setState({ isInitializing: true });
```

Comment:
Do we need to set `isInitializing` here ? Since it's passed down here https://github.com/bigcommerce/checkout-js/pull/700/files#diff-c4c5cfa55abd9305c6b9130693f13101c7eeacfea5e8cf9f4901533dea05aa8eR370 which is a derived prop.

Reply:
We do. This is to activate the loading spinner - https://github.com/bigcommerce/checkout-js/blob/48fb255dfd398722fbc8aca0e100a6f5f9e31f56/src/app/shipping/Shipping.tsx#L128